src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.Tasks.cs

```diff
@@ -356,6 +359,31 @@ internal Task<int> SendToAsync(ArraySegment<byte> buffer, SocketFlags socketFlag
             return tcs.Task;
         }
 
+        public ValueTask SendFileAsync(string? fileName, CancellationToken cancellationToken = default)
```

Comment:
Doc comments are in https://github.com/gfoidl/dotnet-runtime/commit/feade37fbe1c780e50a6b845e1d581d1db14bae7

Will merge them once the rest of the PR is OK to avoid unnecessary CI builds.